website/templates/project/project_header.mako

```diff
@@ -68,8 +81,18 @@
                         % if user['has_read_permissions'] and not node['is_registration'] or (node['is_registration'] and 'admin' in user['permissions']):
                             <li><a href="${node['url']}settings/">Settings</a></li>
                         % endif
-
                     % endif
+                    % if user['can_comment'] or node['has_comments']:
```

Comment:
Is this UI supposed to be visible? I'm not seeing a "Comments" button on a project that I am an admin on.